sdcm/cluster.py

```diff
@@ -1089,40 +1089,41 @@ def install_mgmt(self, scylla_repo, scylla_mgmt_repo, mgmt_port, db_hosts):
         self.log.debug('Install scylla-manager')
         rsa_id_dst = '/tmp/scylla-test'
         mgmt_user = 'scylla-manager'
-        mgmt_conf_tmp = '/tmp/scylla-manager.yaml'
-        mgmt_conf_dst = '/etc/scylla-manager/scylla-manager.yaml'
+        # mgmt_conf_tmp = '/tmp/scylla-manager.yaml'
+        # mgmt_conf_dst = '/etc/scylla-manager/scylla-manager.yaml'
 
         self.remoter.run('sudo yum install -y epel-release', retry=3)
         self.download_scylla_repo(scylla_repo)
         self.download_scylla_manager_repo(scylla_mgmt_repo)
-        if self.is_docker():
+        if self.is_docker(): #TODO: check non-docker env to remove packages as well for manager installation best practice.
             self.remoter.run('sudo yum remove -y scylla scylla-jmx scylla-tools scylla-tools-core'
                              ' scylla-server scylla-conf')
         self.remoter.run('sudo yum install -y scylla-manager')
         if self.is_docker():
             try:
                 self.remoter.run('echo no| sudo scyllamgr_setup')
             except Exception as ex:
+                self.log.warning(ex)
                 pass
         else:
             self.remoter.run('echo yes| sudo scyllamgr_setup')
         self.remoter.send_files(src=self._ssh_login_info['key_file'], dst=rsa_id_dst)
-        self.remoter.run('sudo chmod 0400 {}'.format(rsa_id_dst))
-        self.remoter.run('sudo chown {}:{} {}'.format(mgmt_user, mgmt_user, rsa_id_dst))
-
-        mgmt_conf = {'http': '0.0.0.0:{}'.format(mgmt_port),
-                     'database':
-                         {'hosts': db_hosts,
-                          'timeout': '5s'},
-                     'ssh':
-                         {'user': self._ssh_login_info['user'],
-                          'identity_file': rsa_id_dst}
-                     }
-        (_, conf_file) = tempfile.mkstemp(dir='/tmp')
-        with open(conf_file, 'w') as fd:
-            yaml.dump(mgmt_conf, fd, default_flow_style=False)
-        self.remoter.send_files(src=conf_file, dst=mgmt_conf_tmp)
-        self.remoter.run('sudo cp {} {}'.format(mgmt_conf_tmp, mgmt_conf_dst))
+        # self.remoter.run('sudo chmod 0400 {}'.format(rsa_id_dst))
+        # self.remoter.run('sudo chown {}:{} {}'.format(mgmt_user, mgmt_user, rsa_id_dst))
+
+        # mgmt_conf = {'http': '0.0.0.0:{}'.format(mgmt_port),
+        #              'database':
+        #                  {'hosts': db_hosts,
+        #                   'timeout': '5s'},
+        #              'ssh':
+        #                  {'user': self._ssh_login_info['user'],
+        #                   'identity_file': rsa_id_dst}
+        #              }
+        # (_, conf_file) = tempfile.mkstemp(dir='/tmp')
+        # with open(conf_file, 'w') as fd:
+        #     yaml.dump(mgmt_conf, fd, default_flow_style=False)
+        # self.remoter.send_files(src=conf_file, dst=mgmt_conf_tmp)
+        # self.remoter.run('sudo cp {} {}'.format(mgmt_conf_tmp, mgmt_conf_dst))
```

Comment:
We aren't going to leave these comments like this.

Please move this entire section to helper method, something like "configure_manager".
This should take params and replace them if/when necessary.
It can get its params from the test yaml, fo it'll look like:

if self.params.get('configure_manager', default=False):
    self._configure_manager(...)

Reply:
done